translations/en-us.yml

```diff
@@ -131,8 +131,25 @@ file_detail:
     no_revisions: 'Revisions not available'
     view_tags: 'View tags'
     close_tags: 'Close tags'
+    close_metadata: 'Close metadata'
```

Comment:
We capitalize the second word of the phrase in the lower lines on 138-141; should this be 'Close Metadata' or is the capitalization intentional? One or the other would help consistency.

Reply:
That's intentional. They're used in different contexts, so have different capitalizations.